torch/quantization/fx/quantize.py

```diff
@@ -346,7 +345,7 @@ def _prepare(self, model, qconfig_dict, inplace, prepare_custom_config_dict, is_
 
         # find _inputs_ to matched nodes that are not quantized, these
         # have to be quantized, which requires measuring stats,
-        # initialize an DefaultQuant object for each
+        # initialize an DefaultQuantizeHandler object for each
```

Comment:
The function _prepare is really long, can you refactor it into smaller functions in a later PR?

Reply:
sure